scripts/reducers/flows.reducer.js

```diff
@@ -175,12 +175,15 @@ export default function (state = {}, action) {
       links = mergeLinks(state.unmergedLinks);
     }
 
-    let mapNodeColors = [];
+    let mapNodeColors = [],
+      selectedNodeColors = [];
     if (nodesColoredBySelection && nodesColoredBySelection.length !== 0) {
       //TODO: finish this so that we can color the map too
-      mapNodeColors = getMapColorsFromLinks(links);
+      const colorsFromLinks = getColorsFromLinks(links);
```

Comment:
@nerik what do you think about take the selected node colors from all the links? Is there a better way?